cloudbridge/cloud/providers/aws/resources.py

```diff
@@ -703,7 +694,7 @@ def id(self):
         md5 = hashlib.md5()
         md5.update("{0}-{1}-{2}-{3}".format(
             self.ip_protocol, self.from_port, self.to_port, self.cidr_ip)
-            .encode('ascii'))
```

Comment:
This also looks like it was using proper PEP8 formatting and will now trigger a hanging indent complaint: https://www.python.org/dev/peps/pep-0008/#id3

I'd suggest installing a plugin like flake8 into the editor to do these checks automatically (or manually running it): http://flake8.pycqa.org/en/latest/